sklearn/tests/test_pipeline.py

```diff
@@ -165,20 +165,23 @@ def predict_log_proba(self, X, got_attribute=False):
         return self
 
 
-def test_pipeline_init():
-    # Test the various init parameters of the pipeline.
+def test_pipeline_invalid_parameters():
```

Comment:
Renamed function since we are not validating parameters during instantiation. We decided not to split the function in two because it caused code repetition.